public/js/action-list.js

```diff
@@ -106,6 +117,13 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            // If the item exists, remove it first before reading it
+            if (bottomStatusBar.find('.selected-items').length !== 0) {
+                bottomStatusBar.find('.selected-items').remove();
+            }
+
+            bottomStatusBar.prepend('<span class="selected-items">' + $activeItems.length + ' Items selected</span>');
```

Comment:
The class doesn't exist. Only use classes you'll need. Though, if you're not adding this to the status bar but directly to the footer, the class is required. So.. add it ;)